### PR TITLE
Support flow without OAuth

### PR DESCRIPTION
**Pull request by awegrzyn**
As requested in https://github.com/AliceO2Group/Gui/pull/75

### CODE REVIEW

**Comment by awegrzyn** [comment]
@vladimirkosmala Could you please take a look?

File: Backend/test/mocha-http.js [hunk 1]
```diff
@@ -16,11 +17,24 @@ const token = jwt.generateToken(0, 'test', 1);
...
   });
 
+  it('GET the "/" and return user details', (done) => {
+    http.get('http://localhost:' + config.http.port + '/',
+      (res) => {
+        assert.strictEqual(res.statusCode, 302);
+        const query = url.parse(res.headers.location).query.split('&');
+        assert(query[0].includes('personid'));
+        assert(query[1].includes('name'));
+        assert(query[2].includes('token'));
```
[the remark this answers is not in the document]
Comment (awegrzyn) [reply Backend/test/mocha-http.js]: ok thx, I will change it

[the remark this answers is not in the document]
Comment (awegrzyn) [reply Backend/test/mocha-http.js]: @vladimirkosmala Actually I'm not sure if it's gonna work as this is not URL but location only. It fails with `Uncaught TypeError [ERR_INVALID_URL]`.

[the remark this answers is not in the document]
Comment (awegrzyn) [reply Backend/test/mocha-http.js]: @vladimirkosmala I'll merge it as it is. Could you please create a new PR if you find a more optimal way to solve this?

[the remark this answers is not in the document]
Comment (vladimirkosmala) [reply Backend/test/mocha-http.js]: Sure, I had the same issue on the front, should be quick

**Pull request merged**